test/workers/delete_plain_object_worker_test.rb

```diff
@@ -58,154 +58,6 @@ def test_destroy_method_delete
     end
   end
 
-  class UndestroyableObjectsTest < DeletePlainObjectWorkerTest
-    def setup
-      @object = FactoryBot.create(:service)
-      @object.stubs(:destroyable?).returns(false)
-    end
-
-    attr_reader :object
-
-    def test_perform_destroy_by_association
-      DeletePlainObjectWorker.perform_now(object, %w[HTestClass123 HTestClass1123])
-      System::ErrorReporting.expects(:report_error).never
-      # shouldn't raise anything
-      DeletePlainObjectWorker.perform_now(object, %w[HTestClass123 HTestClass1123])
-      refute object.destroyed?
-    end
-
-    def test_perform_destroy_without_association
-      System::ErrorReporting.expects(:report_error).once.with do |exception, options|
-        exception.is_a?(ActiveRecord::RecordNotDestroyed) \
-          && (parameters = options[:parameters]) \
-          && parameters[:caller_worker_hierarchy] == ['Hierarchy-TestClass-123', "Plain-#{object.class}-#{object.id}"] \
-          && parameters[:error_messages] == ['This product cannot be removed']
-      end
-      # shouldn't raise anything
-      DeletePlainObjectWorker.perform_now(object, %w[Hierarchy-TestClass-123])
-      refute object.destroyed?
-    end
-  end
-
-  class StaleObjectErrorTest < DeletePlainObjectWorkerTest
```

Comment:
This test is not migrated but removed because previously StaleObject errors were handled. Now they are not and normal sidekiq retry is relied on. That should not be a common occurrence like before, because we ensure proper order or delete operations within a single job or a series of jobs.